scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -577,9 +607,8 @@ def __emit_report_if_necessary(self, current_time=None):
             self.__last_report_time = current_time
 
             warm_attempts_info = ''
-            for category in ['total', 'success', 'temp_error', 'perm_error', 'unknown_error', 'unhandled_error']:
-                current_amount = self.__warming_attempts.get(category, 0)
-                previous_amount = self.__last_reported_warming_attempts.get(category, 0)
+            stats = __gather_report_stats()
```

Comment:
missing `self.`

Reply:
Gah.  I had it in my working tree, just not committed 🤦‍♂️